robosuite/robots/mobile_robot.py

```diff
@@ -266,6 +266,15 @@ def setup_references(self):
             self.sim.model.joint_name2id(joint) for joint in self.robot_model.head_joints
         ]
 
+        self._ref_actuator_to_joint_id = np.ones(self.sim.model.nu).astype(np.int32) * (-1)
+        for part_name, actuator_ids in self._ref_actuators_indexes_dict.items():
+            self._ref_actuator_to_joint_id[actuator_ids] = np.array(
```

Comment:
add a very small comment in the code explaining how this mapping works (conceptually at a high level)